conda/recipes/meta.yaml

```diff
@@ -18,15 +16,19 @@ source:
 
 build:
   number: {{ git_revision_count }}
+  noarch: python
   script: python -m pip install . -vvv
 
 
 requirements:
+  build:
+    - python
+    - setuptools
   run:
-  {% for req in setup_py_data.get('install_requires', []) %}
-    - {{ req }}
-  {% endfor %}
     - python
+    {% for req in setup_py_data.get('install_requires', []) %}
+    - {{ req }}
+    {% endfor %}
     - tensorflow
```

Comment:
wondering if we should remove the dependency on tensorflow and nvtabular in the conda package since they're optional dependencies in the pip package

Reply:
removed to match the way the pip package dependencies work